objects/obj_electric_floor/Step_0.gml

```diff
@@ -1,17 +1,24 @@
 /// @description Check Player Collision
 var damage = 8;
 
-if (place_meeting(x,y-1,obj_player) and power_on){
+var player_instance = instance_nearest(x,y,obj_player);
```

Comment:
There will only ever be one player object existing at a time, so I'm pretty sure instance_nearest is unnecessary. Replace every "player_instance" with just obj_player

Reply:
Is this always going to be the case, I know in some of the rooms we have now there are multiple player instances like the cannon player or whatnot. I guess my concern would be the plan of implementing the different players if we have skins or whatnot.

Reply:
I think it will most likely always be the case, I know we aren't planning to use the cannon player anymore so we won't have to worry about that. Also, if we do implement skins they will ideally still be implemented within the obj_player object. It's a pretty small nitpicky thing so I'm gonna go ahead and approve it anyways. Everything else looks good!